src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -266,10 +266,10 @@ class TypeSpecTest {
         |import kotlin.collections.Map
         |
         |interface Service {
-        |  @Headers([
+        |  @Headers(
         |      "Accept: application/json",
         |      "User-Agent: foobar"
-        |  ])
+        |  )
```

Comment:
This seems like a regression

Reply:
It's not. Since the param is `vararg value: String` it's fine to pass strings vararg-style.

Reply:
Do we still have tests which exercise the annotation literal syntax then?

Reply:
Yes, and I added a few as well:

```kotlin
@Test fun valueAndOtherVarargParameterEmitNames() {
  val annotationSpec = AnnotationSpec.builder(ValueAndOtherVarargParam::class)
      .addMember("a", CodeBlock.of("%S", "a"), emitName = true)
      .addMember("a", CodeBlock.of("%S", "b"), emitName = true)
      .addMember("a", CodeBlock.of("%S", "c"), emitName = true)
      .addMember("value", "1", emitName = true)
      .build()

  assertThat(toString(annotationSpec)).isEqualTo("""
    |package com.squareup.tacos
    |
    |import com.squareup.kotlinpoet.AnnotationSpecTest
    |
    |@AnnotationSpecTest.ValueAndOtherVarargParam(
    |    a = [
    |        "a",
    |        "b",
    |        "c"
    |    ],
    |    value = 1
    |)
    |class Taco
    |""".trimMargin())
}
```